cardano-cli/src/Cardano/CLI/EraBased/Run/Query.hs

```diff
@@ -1035,7 +1035,9 @@ filteredUTxOsToText :: Api.ShelleyBasedEra era -> UTxO era -> Text
 filteredUTxOsToText sbe (UTxO utxo) = do
   mconcat
     [ title
+    , "\n"
```

Comment:
You can use https://hackage.haskell.org/package/base-4.19.1.0/docs/Prelude.html#v:unlines